internal/data/services.go

```diff
@@ -0,0 +1,22 @@
+package data
+
+import (
+	"database/sql"
+	"errors"
+)
+
+var (
+	ErrRecordNotFound = errors.New("record not found")
+)
+
+type Services struct {
+	Talks    TalkService
+	TimeZone TimeZoneService
+}
+
+func NewServices(db *sql.DB) Services {
+	return Services{
```

Comment:
I usually like to make my new functions return an error too, and check that db is not nil. Adds a lilt more safety.